webui/src/components/user.jsx

```diff
@@ -141,7 +141,7 @@ export const UserProfile = React.createClass({
                     </div>
                     <div className="row">
                         <h3>Own drafts</h3>
-                            <p><Link to={"/records?drafts=1&q=publication_state%3Adraft"}>
```

Comment:
This one is in fact useful otherwise you will also have published drafts.
We can also change the rest API to not show published drafts.

Reply:
I'm adding it back only when it's sent to the server, here: https://github.com/EUDAT-B2SHARE/b2share/pull/1602/files#diff-4202620782d33deec0c6acdb6ad427d6R523

Reply:
Strange I get published drafts too when I test it.